spec/features/create_object_h2_spec.rb

```diff
@@ -71,7 +71,7 @@
     # Checks if title is on resulting display
     expect(page).to have_content(item_title)
     # This happens asynchronously, it might take a bit
-    expect(page).to have_content('https://sul-purl-stage.stanford.edu')
+    expect(page).to have_content(/https:\/\/(sul-|)purl-(stage|qa)\.stanford\.edu/)
```

Comment:
Should this be a config variable rather than a regex?

Reply:
I think it would need to be what is in h2 shared_configs, which may change - kind of a config of a config.  The above works now, and right now qa is set to stage ... so it should be future-proof?

Reply:
I guess what I'm saying is `expect(page).to have_content Settings.purl_link` is easier to understand

Reply:
ok, added it to settings and clarified where it was coming from with comments.   I understood you the first time but was concerned the value in qa.yml  would be puzzling to get right.